tasmota/xdrv_23_zigbee_A_impl.ino

```diff
@@ -1331,6 +1331,36 @@ extern "C" {
       return 1;
     }
   }
+
+  void convert_seconds_to_dhms(uint32_t n, char *result){
```

Comment:
pleas add a length to the buffer as a protection:
`void convert_seconds_to_dhms(uint32_t n, char *result, size_t result_len) {`